CMakeLists.txt

```diff
@@ -213,6 +225,8 @@ if (ENABLE_MPI)
     message("-- MPI include dirs: ${MPI_CXX_INCLUDE_DIRS}")
     message("-- MPI libraries:    ${MPI_CXX_LIBRARIES}")
     message("-- MPI link flags:   ${MPI_CXX_LINK_FLAGS}")
+    message("-- MPI executable:   ${MPIEXEC_EXECUTABLE}")
+    message("-- MPI numproc flag: ${MPIEXEC_NUMPROC_FLAG}")
 
     target_include_directories(libocca PRIVATE ${MPI_CXX_INCLUDE_DIRS})
     target_link_libraries(libocca PRIVATE ${MPI_CXX_LIBRARIES})
```

Comment:
Since we're not explicitly passing the MPI includes and libs to the MPI examples in this PR, we should likely change these PRIVATE includes and libs to PUBLIC so they are propagated to targets. Otherwise theres a change the mpi.h and mpi libs aren't found in the examples.

Reply:
This has not been a problem for my builds so far, but better safe than sorry.

Reply:
Done.